src/mw/device/include/madrona/state.hpp

```diff
@@ -221,6 +233,7 @@ class StateManager {
         
         // The size of this array corresponds to the number of worlds
         int32_t *sortOffsets;
```

Comment:
This would be slightly annoying but might be worth renaming to worldOffsets and worldCounts respectively. Don't have to though.